src/work-stealing-queue.h

```diff
@@ -36,6 +36,13 @@ static inline ws_array_t *create_ws_array(size_t capacity, int32_t eltsz) JL_NOT
 
 typedef struct {
     _Atomic(int64_t) top;
+    // put `top` and `bottom` on different cache lines
```

Comment:
`__attribute__((aligned(n by by)))` could also be used, right?

Also, with the other method, maybe nicer to use `char *` for clarity?

Reply:
Should be `alignas` https://en.cppreference.com/w/c/language/_Alignas and I think `JL_CACHE_BYTE_ALIGNMENT` is the constant that is supposed to be cache-line.

Reply:
Could we somehow use https://en.cppreference.com/w/cpp/thread/hardware_destructive_interference_size

Reply:
I was looking at that, but it's C++ only

Reply:
It's also not implemented by clang